core/src/main/scala/ammrunner/AmmoniteFetcher.scala

```diff
@@ -59,23 +59,23 @@ import scala.io.{BufferedSource, Codec}
         if (interpOnly)
           "ammonite-interp-api_" + versions.scalaVersion
         else
-          "ammonite-repl-api_" + versions.scalaVersion,
+          "ammonite-repl-api_" + twoThreeInterop(versions.scalaVersion),
         versions.ammoniteVersion
       )
       Seq(api) ++ compilerDeps
     }
 
+    def twoThreeInterop(scalaVersion: String) = {
+      if (scalaVersion.startsWith("3")) BuildInfo.scala213
+      else scalaVersion
+    }
+
     def createFetcher(): Fetch = {
       val cache = Cache.create()
       if (progressBars)
         cache.withLogger(Logger.progressBars())
       val fetch = Fetch.create()
         .withCache(cache)
-        .withResolutionParams(
-          // FIXME This mutates resolutionParams in place
-          resolutionParams
-            .withScalaVersion(versions.scalaVersion)
```

Comment:
This caused issues since in Scala 3 it was forcing 3 on all the artifacts, which is problematic since there is a bunch of 2.13 on the classpath for `cs resolve com.lihaoyi:ammonite_3.0.2:2.4.1`. However, I'm not sure if just fully removing this is ok.

Reply:
It might be ok to remove it… But the `3.x` version changing the `_2.13*` suffixes could be fixed in coursier too.